conda/recipes/cucim/meta.yaml

```diff
@@ -38,21 +38,33 @@ requirements:
   build:
     - {{ compiler("c") }}
     - {{ compiler("cxx") }}
+    {% if cuda_major == "11" %}
+    - {{ compiler('cuda11') }} ={{ cuda_version }}
+    {% else %}
+    - {{ compiler('cuda') }}
+    {% endif %}
+    - cuda-version ={{ cuda_version }}
     - cmake >=3.23.1,!=3.25.0
     - make
     - ninja
     - sysroot_{{ target_platform }} {{ sysroot_version }}
   host:
     - click
-    - cudatoolkit ={{ cuda_version }}
+    - cuda-version ={{ cuda_version }}
+    {% if cuda_major == "11" %}
+    - cudatoolkit
+    {% endif %}
```

Comment:
I wonder if this is needed anymore.
```suggestion
```

Reply:
Yeah that's a good point. I don't think so. Let's try dropping it